packages/suite/src/views/wallet/receive/components/FreshAddress/index.tsx

```diff
@@ -92,7 +114,7 @@ const FreshAddress = ({
             {addressPath && <AddressPath>{addressPath}</AddressPath>}
             <AddressContainer>
                 <StyledInput
-                    label={<Translation id={addressLabel} />}
+                    label={<InputLabel isBitcoin symbol={account.symbol} />}
```

Comment:
`isBitcoin` here will always be true. Not sure but I guess you want to pass the value declared higher up?

Reply:
good catch, fixed